audio-clips/src/main/java/org/odk/collect/audioclips/AudioClipViewModel.kt

```diff
@@ -20,7 +21,7 @@ class AudioClipViewModel(private val mediaPlayerFactory: Supplier<MediaPlayer>,
     private var _mediaPlayer: MediaPlayer? = null
 
     private val currentlyPlaying = MutableLiveData<CurrentlyPlaying?>(null)
-    private val error = MutableLiveData<Exception?>()
+    private val error = MutableLiveData<Consumable<Exception>?>()
```

Comment:
It doesn't need to be nullable now as it uses `Consumable` right? We used null to make the errors one-time occurrences.

Reply:
Ah yeah I guess we never set the error back to `null` now. It feels like the `error` should still be `null` once there's a successful load clip load though. I'll tweak that.